packages/docs/docs/guides/docsqa.md

```diff
@@ -86,7 +86,7 @@ flowchart LR
   que[Query] -->|string| embed2[Embed] -->|vector| vdb2[(Vector DB)] -->|similar chunks| LLM
 ```
 
-If you use the built-in DocsQA tag from AI.JSX, then you just need to decide how to present the chunk to your LLM:
+If you use the built-in `DocsQA` component from AI.JSX, then you just need to decide how to present the chunk to your LLM:
```

Comment:
Nit: you can link to the document page (not necessary if it's been done before).

Please check the format, I haven't checked it myself.
```suggestion
If you use the built-in [`DocsQA`](../api/modules/batteries_docs#docsqa) component from AI.JSX, then you just need to decide how to present the chunk to your LLM:
```